### PR TITLE
refactor: add `@throws RedirectException` in `Controller::initController`

### PR DESCRIPTION
**Pull request by paulbalandan**
**Description**
The underlying `force_https` may throw `RedirectException`

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/Controller.php [hunk 1]
```diff
@@ -78,6 +79,7 @@ class Controller
      * @return void
      *
      * @throws HTTPException
+     * @throws RedirectException
```
Comment (michalsn) [inline system/Controller.php:82]: Why not a one-liner?

Comment (paulbalandan) [reply system/Controller.php:82]: What do you mean?

Comment (michalsn) [reply system/Controller.php:82]: Something like this:
```php
@throws HTTPException|RedirectException
```

Comment (paulbalandan) [reply system/Controller.php]: fixed

Comment (michalsn) [reply system/Controller.php]: Thanks!

**Review by michalsn — APPROVED** [review]

**Pull request merged**